contracts/agent-router/verify-live-capacity.sh

```
@@ -0,0 +1,697 @@
+
+# The offline-restore and cached-contrast checks stop the real producer and the
+# real edge container mid-check. An interrupt during those windows would leave
+# production degraded (it happened: a run killed mid-offline-check left
+# edge-heartbeat.service stopped until noticed). Restore both on the way out of
+# ANY abnormal exit; idempotent when nothing was stopped.
+restore_production() {
+    if [ "$EDGE_LOCAL" = 1 ] || [ -n "$EDGE_SSH" ]; then
+        ssh_edge "systemctl --user start edge-heartbeat.service" >/dev/null 2>&1 || true
+        ssh_edge "docker start edge-llama-swap" >/dev/null 2>&1 || true
+    fi
+}
```

Shellcheck (0.11.0)

[info] This function is never invoked. Check usage (or ignored if invoked indirectly).

(SC2329)